pennylane/gradients/pulse_gradient.py

```diff
@@ -604,8 +657,60 @@ def _generate_tapes_and_cjacs(tape, idx, key, num_split_times, use_broadcasting)
         for tau in taus:
             split_evolve_ops, psr_coeffs = _split_evol_ops(op, ob, tau)
             tapes.extend(_split_evol_tape(tape, split_evolve_ops, op_idx))
-    avg_prefactor = (t1 - t0) / num_split_times
-    return cjacs, tapes, avg_prefactor, psr_coeffs
+    int_prefactor = (t1 - t0) / num_split_times
+    return cjacs, tapes, int_prefactor, psr_coeffs
+
+
+def _tapes_data_hardware(tape, op_id, key, num_split_times, use_broadcasting):
```

Comment:
Minor: could you type hint or document the arguments of the private functions, it is not really clear what we expect for each of them.

Reply:
Done.